addons/mail/static/src/models/thread/thread.js

```diff
@@ -2280,6 +2294,17 @@ function factory(dependencies) {
         serverMessageUnreadCounter: attr({
             default: 0,
         }),
+        /**
+         * Not a real field, used to trigger `thread.markAsSeen` when
+         * notification "Handled by Email".
+         */
+        shouldBeSetAsSeen: attr({
```

Comment:
Not sure if this name is appropriate, it gives the impression that the field contains a boolean, which is not the case.